src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -280,6 +280,22 @@ s! {
         pub sem_flg: c_short,
     }
 
+    pub struct input_event {
+        pub time: crate::timeval,
+        pub type_: crate::u_short,
+        pub code: crate::u_short,
+        pub value: c_int,
+    }
+
+    pub struct input_absinfo {
+        pub value: c_int,
+        pub minimum: c_int,
+        pub maximum: c_int,
+        pub fuzz: c_int,
+        pub flat: c_int,
+        pub resolution: c_int,
+    }
```

Comment:
The integers can use `i32` to match https://github.com/freebsd/freebsd-src/blob/38b67578fb4bbf568f7012ca3921a4d15cfe7c5d/sys/dev/evdev/input.h#L51-L65